### ISSUE TITLE
RPN and RCNN checkpoints

### ISSUE THREAD

**Issue by swdev1202** [Kartik17/PointRCNN-Argoverse#1]
Hi @Kartik17!
First of all, thank you for your work :)

I was running the quick demo and saw `rpn_ckpt.pth` and `rcnn_ckpt.pth` .
Are they already pre-trained with Argoverse data?
If so, I get pretty bad results on train/val sets.
May I know your evaluation results on validation sets?

Thank you!

**Comment by Kartik17** [comment]
Yeah, it's trained on Argoverse dataset, how bad were your results ?

**Comment by gebawe** [comment]
Hi @Kartik17 

I am also working on the Argoverse datasets, and I am interested in your work PointRCNN.
I have tried it both with the pre-trained weights "rpn_ckpt.pth and rcnn_ckpt.pth" and also by training it from scratch. but the result is really low ~0.5 recall at 0.1 IoU.
May I know your evaluation results on validation sets?  @swdev1202
Can you recommend to me some configuration parameters?

**Comment by Kartik17** [comment]
Hi, @awethaileslassie I will check again, I haven't worked on this repo for a while.

**Comment by gebawe** [comment]
@Kartik17 Any updates?
